de-lib/src/main/java/org/iplantc/de/client/models/notifications/NotificationCategory.java

```diff
@@ -16,7 +16,7 @@
     /** Analysis notifications */
     ANALYSIS("Analysis"),
     /** tool rquest status update notification */
-    TOOLREQUEST("Tool Request"),
+    TOOL_REQUEST("Tool Request"),
```

Comment:
I noticed these notification category types have changed from no spaces to having underlines, so I updated them.  
The only little snaggling bug right now is that Tool Request notifications show up as having no message body in the DE UI for some reason.  Will look into this after the break if nobody else has.